web-ts/src/routes/categories/CategoriesPage.tsx

```diff
@@ -1,6 +1,10 @@
-import React from 'react'
```

Comment:
I think we can flatten this structure to `src/routes/CategoriesPage.tsx`.

Reply:
Sure, but lets just wait for https://issues.tuerantuer.org/browse/IGAPP-640 as I did not create this in this PR.

Reply:
Created an issue: https://issues.tuerantuer.org/browse/IGAPP-671